code/workspaces/infrastructure-api/src/notifications/notificationsController.js

```diff
@@ -0,0 +1,64 @@
+import { matchedData } from 'express-validator';
+import axios from 'axios';
+import notificationsRepository from './notificationsRepository';
+import emailNotificiationAdapter from './adapters/emailNotificationAdapter';
+import config from '../config/config';
+
+const adapters = [
+  emailNotificiationAdapter,
+];
+
+async function sendNotification(request, response, next) {
+  const { title, message, userIDs } = matchedData(request);
+
+  try {
+    const authUrl = config.get('authorisationService');
+    // translate the userIDs to emails so we only store anonymous data but use emails when exiting the system
+    const userEmails = await Promise.all(userIDs.split(',').map(async (id) => {
+      const r = await axios.get(`${authUrl}/users/${id}`, { headers: { authorization: request.headers.authorization } });
+      const userInfo = r.data;
+      return userInfo.name;
```

Comment:
Can we assume that this will always return an email for a given user ID?

Reply:
From the stand up discussion, we've agreed we're going to go with this for now and add a new story to clarify how user's emails are handled in the system.